src/openrct2/Context.cpp

```diff
@@ -108,6 +109,9 @@ namespace OpenRCT2
 #ifdef ENABLE_SCRIPTING
         ScriptEngine _scriptEngine;
 #endif
+#ifndef DISABLE_NETWORK
+        NetworkBase _network;
+#endif
```

Comment:
Can't we just have some null network so this isn't required

Reply:
Its on my list for sure, I never got quite to the point of splitting up the three classes and adding a dummy.

Reply:
Dummy would be great for not having all those guards, indeed :)